README.md

```diff
@@ -246,168 +246,188 @@ crawler.on("fetchcomplete", function(queueItem, data, res) {
+    The user agent the crawler will report.
+* `crawler.decodeResponses=false` -
     The response bodies will be intelligently character converted to standard
-    JavaScript strings using the [`iconv-lite`](https://www.npmjs.com/package/iconv-lite)
-    module. The character encoding is interpreted from the Content-Type header
-    firstly, and secondly from any &lt;meta charset="xxx" /&gt; tags.
-*    `crawler.respectRobotsTxt` -
+    JavaScript strings using the
+    [iconv-lite](https://www.npmjs.com/package/iconv-lite) module. The character
+    encoding is interpreted from the Content-Type header firstly, and secondly
+    from any `&lt;meta charset="xxx" /&gt;` tags.
+* `crawler.respectRobotsTxt=true` -
     Controls whether the crawler should respect rules in robots.txt (if such a
-    file is present). True by default. The [`robots-parser`](https://www.npmjs.com/package/robots-parser)
-    module is used to do the actual parsing.
-*    `crawler.queue` -
+    file is present). The
+    [robots-parser](https://www.npmjs.com/package/robots-parser) module is used
+    to do the actual parsing.
+* `crawler.queue` -
     The queue in use by the crawler (Must implement the `FetchQueue` interface)
-*   `crawler.allowInitialDomainChange` -
-    If the response for the initial url is a redirect to another domain
-    (e.g. from github.net to github.com), update `crawler.host` to
-    continue the crawling on that domain. Defaults to false.
-*    `crawler.filterByDomain` -
+*   `crawler.allowInitialDomainChange=false` -
+    If the response for the initial URL is a redirect to another domain (e.g.
+    from github.net to github.com), update `crawler.host` to continue the
+    crawling on that domain.
+* `crawler.filterByDomain=true` -
     Specifies whether the crawler will restrict queued requests to a given
     domain/domains.
-*    `crawler.scanSubdomains` -
-    Enables scanning subdomains (other than www) as well as the specified domain.
-    Defaults to false.
-*    `crawler.ignoreWWWDomain` -
+* `crawler.scanSubdomains=false` -
+    Enables scanning subdomains (other than www) as well as the specified
+    domain.
+* `crawler.ignoreWWWDomain=true` -
     Treats the `www` domain the same as the originally specified domain.
-    Defaults to true.
-*    `crawler.stripWWWDomain` -
+* `crawler.stripWWWDomain=false` -
     Or go even further and strip WWW subdomain from requests altogether!
-*    `crawler.stripQuerystring` -
-    Specify to strip querystring parameters from URLs. Defaults to false.
-*    `crawler.discoverResources` -
-    Use simplecrawler's internal resource discovery function. You can replace it
-    with your own function, which must accept a buffer and a queueItem, and add
-    the discovered resources to the crawler queue:
+* `crawler.stripQuerystring=false` -
+    Specify to strip querystring parameters from URL's.
+* `crawler.discoverResources` -
+    simplecrawler's built-in discovery function is purposefully naïve. The
```

Comment:
I wonder if we might create a dedicated in the section to talk about link discovery, and link to it from here?